data_set_distances.go

```diff
@@ -0,0 +1,99 @@
+//
```

Comment:
Can you remove these comments, they show up in the generated documentation:

https://godoc.org/github.com/montanaflynn/stats

The commit history and contribution is stored in the .git and GitHub.